x-pack/plugins/alerting/common/alert_summary.ts

```diff
@@ -8,6 +8,12 @@
 export type RuleStatusValues = 'OK' | 'Active' | 'Error';
 export type AlertStatusValues = 'OK' | 'Active';
 
+export interface ExecutionDuration {
+  average: number;
+  values: number[];
```

Comment:
Do we need both `values` and `valuesWithTimestamp`? Looks like we could remove `values`?

Reply:
I will check for the usage of this.

Reply:
Removed in the commit https://github.com/elastic/kibana/pull/118978/commits/d5f9d5edb4fce8b237d55be682746b509f307edb